examples/gfx/opengl/5_opengl_flappy_bird/main.cpp

```diff
@@ -27,8 +27,12 @@ int main() {
     glEnable(GL_BLEND);
     glBlendFunc(GL_SRC_ALPHA, GL_ONE_MINUS_SRC_ALPHA);
 
-    auto input = window->get_input();
-    Game game(screen_width, screen_height, input);
+    InputHandler input_handler = InputHandler();

```

Comment:
Same as above

Reply:
That's my code style. I don't think it harms our code in any way. For me, the `auto` keyword reduces readability. I don't like to use it, except when there is `std::shared_pointer`.